### DISCUSSION TITLE
On the mass of benzene

### DISCUSSION THREAD

**Discussion by dspoel** [ricalessandri/Martini3-small-molecules#29] [Q&A] [answered]
Not sure whether this is a bug or a feature, but the mass of a benzene molecule is 108 Da, rather than 78 Da.

**Comment by ricalessandri** [comment, marked as answer]
By default Martini (3) uses standard masses for the beads: 36 Da for tiny beads, 54 Da for small beads, 72 Da for regular beads. Hence the masses you see in the benzene itp.

**Comment by dspoel** [comment]
Just wondering, since the density of benzene liquid is now 1300 g/l rather than 900 g/l...

**Comment by ricalessandri** [comment]
To compute the density then you should use the actual mass of benzene, either by generating a new `.tpr` with the right mass and the `gmx mdrun -rerun` (if you want to use `gmx density`) or via the box volume, etc.
